localstack/services/s3/s3_listener.py

```diff
@@ -634,7 +634,7 @@ def strip_chunk_signatures(data):
 
 
 def is_bucket_available(bucket_name):
-    body = {}
+    body = {''}
```

Comment:
What is the purpose of this change? Note that `body` should be of type `dict`, not `set`. With your change, line 637 would create a `set` with a single item in it..?

Reply:
Ok will fix it